integration_tests/return_spec.json

```diff
@@ -0,0 +1,51 @@
+{
+  "version": 1,
```

Comment:
Why do we need a return_spec? Thanks.

Reply:
This is the interface which the network runners need to implement in order to interact with Lucy successfully